.github/workflows/release.yml

```diff
@@ -51,22 +51,10 @@ jobs:
           } | tee -a $GITHUB_ENV
           echo $releaseChangelog
 
-          # Check if version is a pre-release
-          preRelease=$((`echo $releaseVersion | cut -d '.' -f2` % 2))
-          {
-            echo -n "preRelease="
-            if [ $preRelease = 1 ] || [ $preRelease = "true" ]; then
-              echo "true";
-            else
-              echo "false";
-            fi
-          } | tee -a $GITHUB_ENV
-          echo $preRelease
-
       - name: Create release
         uses: ncipollo/release-action@v1.15.0
         with:
           tag: ${{ env.releaseVersion }}
           body: ${{ env.releaseChangelog }}
-          prerelease: ${{ env.preRelease }}
+          prerelease: false
```

Comment:
According to `https://github.com/ncipollo/release-action` the default value is `false` so maybe drop this line entirely?